rbd-target-api.py

```diff
@@ -992,11 +992,11 @@ def disk(pool, image):
     **RESTRICTED**
     Examples:
     curl --insecure --user admin:admin -d mode=create -d size=1g -d pool=rbd -d count=5
-        -X PUT https://192.168.122.69:5000/api/disk/rbd.new2_
-    curl --insecure --user admin:admin -d mode=create -d size=10g -d pool=rbd
-        -X PUT https://192.168.122.69:5000/api/disk/rbd.new3_
-    curl --insecure --user admin:admin -X GET https://192.168.122.69:5000/api/disk/rbd.new2_1
-    curl --insecure --user admin:admin -X DELETE https://192.168.122.69:5000/api/disk/rbd.new2_1
+        -X PUT https://192.168.122.69:5000/api/disk/rbd/new0
```

Comment:
Nit: I think the rationale here was that `count=5` so it's going to create 5 disks and append the index at the end of the image name

Reply:
Okay, will add it back.